include/SDL3/SDL_begin_code.h

```diff
@@ -389,7 +389,9 @@
 #endif /* SDL_FORCE_INLINE not defined */
 
 #ifndef SDL_NORETURN
-#ifdef __GNUC__
+#if defined (SDL_PLATFORM_NGAGE)
+#define SDL_NORETURN
+#elif defined(__GNUC__)
```

Comment:
If this is something that's only available in newer GCC, it might be best to add a version check here so that other platforms can benefit.

Reply:
Done.